pegasus/sites.v3/code.org/public/alternative-classrooms.md.partial

```diff
@@ -200,6 +200,8 @@ Click each question to expand and see the answers. Don’t see your question ans
 <div style="text-align:center">
 <img src="/images/athome/unescovid19.png" style="width:35%">
 </div>
+<<<<<<< HEAD
```

Comment:
remove this? :)

Reply:
!!! thank you! don't know how I missed that!